packages/components/tree/src/tree-node.vue

```diff
@@ -184,6 +186,23 @@ export default defineComponent({
       return getNodeKeyUtil(tree.props.nodeKey, node.data)
     }
 
+    const getNodeClass = (node: Node) => {
+      const nodeClassFunc = props.props.class
+      let className = nodeClassFunc
+      if (nodeClassFunc instanceof Function) {
```

Comment:
**Suggestion:** you can use `isFunction` exported from `@vue/shared` for this judgment.